setup-old-fashioned

```diff
@@ -93,8 +93,8 @@ def ubuntu_distro_info(options):
 
 
 if __name__ == '__main__':
-    if '16.04' not in platform.linux_distribution():
-        print("Old-fashioned must be run on Xenial Ubuntu.")
+    if '18.04' not in platform.linux_distribution():
+        print("Old-fashioned must be run on Bionic Ubuntu.")
```

Comment:
I'd consider retaining optional support for xenial for a while.  It seems as though it might smooth the migration path for users, and we aren't going to be dropping support for it for at least a year or two.  (Does ubuntu-old-fashioned support being run on powerpc?)

Reply:
old-fashioned only supports amd64 and arm64 (primarily because we don't have a backend to build other architectures).

I can re-enable xenial support :-)